### ISSUE TITLE
Back Button handling on Android 

### ISSUE THREAD

**Issue by MarcusRiemer** [josefarias/hotwire_combobox#39]
I just double checked for Android and both Firefox and Chrome differ in one important aspect when dealing with `<select>`: Hitting the OS back button closes the dialogue. With this gem it instead navigates back, possibly leading to lost input data.

**Comment by josefarias** [comment]
Good catch! Thanks. Will look into this. I want to fix some of the other issues first.
